src/components/DatePicker/DatePicker.scss

```diff
@@ -28,29 +25,41 @@ $in-range-border-color: #9ca6de;
   margin-left: spacing();
   max-width: calc(100% - #{spacing()});
   min-width: rem(230px);
+  table-layout: fixed;
```

Comment:
```suggestion
  width: 100%;
  table-layout: fixed;
```

I think you might need to stretch the container by default for FireFox.

![Screen Shot 2020-09-23 at 1 33 28 PM](https://user-images.githubusercontent.com/19199063/94048557-b109e880-fda1-11ea-98e6-acaf658089b8.png)